addons/account/models/account_report.py

```diff
@@ -301,6 +302,9 @@ class AccountReportLine(models.Model):
     _order = 'sequence, id'
 
     name = fields.Char(string="Name", translate=True, required=True)
+    json_name = fields.Json(string="Technical Field for the ",
```

Comment:
Too bad we need a new field for that ; couldn't we do something on the framework side that would just allow directly giving the translations ?

Reply:
@oco-odoo We could overwrite the write/create methods to have fields like name@fr e.g.   (We could do it in the field_to_sql as well, but then maybe we should change in the csv as well to a . instead of a @: see https://github.com/odoo/odoo/pull/186158)